docs/bin/import-nifi-readme.sh

```diff
@@ -8,15 +8,70 @@
 # have to hand-edit the files after this import!
```

Comment:
Yes. I've tried to minimize the number of conversions that need to be hand-edited, but some are difficult to get rid of. For example, some links in the tutorial and nifi README.md files are manual links to geomesa.org or relative links to other READMEs that should be converted to ``:ref:``, ``:doc:``, or ``:download:`` directives.